pkglistgen/tool.py

```diff
@@ -562,6 +592,7 @@ def solve_project(
                         module.solved_packages[arch].pop(p, None)
 
         self._collect_unsorted_packages(modules, self.groups.get('unsorted'))
+        self.write_productcompose()
```

Comment:
This is called even if `000productcompose` doesn't exist or is `None`

Reply:
will fix that